lib/plines/step.rb

```diff
@@ -65,6 +65,10 @@ def fan_out(&block)
       fan_out_blocks << block
     end
 
+    def run_if?(&block)
```

Comment:
Done. I think, one is enough. It reflects intention (`fan_out` is implementation detail), and is unambiguous. Too many helpers hurt.